cli/src/command/run_dev.rs

```diff
@@ -34,9 +34,14 @@ use crate::command::{
 #[derive(StructOpt, Debug, Serialize, Deserialize)]
 #[non_exhaustive]
 pub struct RunDev {
-    /// The path where configuration and db files will be generated.
-    #[structopt(long, short = "a")]
-    pub artifacts_dir: PathBuf,
+    /// Path to a directory for blockchain database and configuration files.
+    ///
+    /// Database is located in <blockchain_path>/db directory, node configuration files
+    /// are located in <blockchain_path>/config directory. Existing files and directories are
+    /// reused. To generate new node configuration and start a new blockchain, the user must
+    /// use --clean flag or specify an another directory.
+    #[structopt(long)]
```

Comment:
What do you think about to add a short name for this param? For example `-p` or something like this ? I suppose it'd be convenient since `run-dev` command uses often in manual mode.

Reply:
Probably makes sense